packages/riipen-ui/src/components/RadioButtonGroup.jsx

```diff
@@ -0,0 +1,128 @@
+import css from "styled-jsx/css";
+import PropTypes from "prop-types";
+import React from "react";
+
+import clsx from "clsx";
+
+import withClasses from "../utils/withClasses";
+
+import InputLabel from "./InputLabel";
+import Typography from "./Typography";
+
+const RadioButtonGroup = ({
+  children,
+  error,
+  hint,
+  label,
+  onChange,
+  required,
+  warning,
+  ...other
+}) => {
+  const linkedStyles = css.resolve`
+    .radioButtons .radioButton:first-child {
+      border-top-left-radius: 15px;
+      border-bottom-left-radius: 15px;
```

Comment:
I wounder if we should put these border radius sizes into our theme? `shape.borderRadius.xxl`? Your call. It's sort of a one off.

Reply:
there is no standardized set of radii so I am going to opt to leaving it out for now, if we get a set to use then I am all for this change

Reply:
I dont think we have very many rounded edges in the UI Kit at the moment